docs/content/routing/services/index.md

```diff
@@ -677,7 +677,7 @@ metadata:
   name: myca
 
   data:
-    tls.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0=
+    ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0=
```

Comment:
FYI, I see in the code that we apparently use whatever's in data (as long as there's only one field) anyway, so it seems that the existing example would have worked nonetheless. And there's a comment saying we're supposed to drop that behaviour at some point.
But yeah, better be matching the doc.